gems/ruby-saml/CVE-2016-5697.yml

```diff
@@ -0,0 +1,10 @@
+---
+gem: ruby-saml
+framework: rails
+cve: 2016-5697
+url: https://github.com/onelogin/ruby-saml
+title: XML Signature wrapping attack
+date: 2016-06-24
+description: Ruby-saml prior to version 1.3.0 is vulnerable to an XML signature wrapping attack. Ruby-saml users must update to 1.3.0 version which implements 3 extra validations to mitigate this kind of attack.
+cvss_v3: 6.1
```

Comment:
Actually, you can leave this for now. I think it will be fine (just not officially supported). I filed #262 to add this to the schema.